src/main/java/core/basesyntax/Calculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculate {
```

Comment:
According to the checklist, avoid using verbs for class/interface names. Consider renaming the interface to something that represents an entity or a concept rather than an action, such as 'AreaCalculable' or 'AreaCalculator'.